test/visualize/test.mlir

```diff
@@ -0,0 +1,105 @@
+// Test cases for FusePatternsPass
+// Tests both FAddFAdd and FMulFAdd fusion patterns
+
+// Test 1: Basic FAddFAdd fusion (LHS fadd)
+// RUN: mlir-neura-opt --assign-accelerator --lower-arith-to-neura --fuse-patterns %s | FileCheck %s --check-prefix=CHECK-FADD-FADD-LHS
+
```

Comment:
Your png is generated by hand, right?

Could the commands, i.e.,
```
// RUN: mlir-opt --allow-unregistered-dialect --view-op-graph test_fused.mlir 2> %t-test.dot
// RUN: FileCheck %s --input-file=%t-test.dot --check-prefix=VIS
// RUN: dot -Tpng %t-test.dot -o %t-test.png
```
be added here?

And correspondingly, add:
```
CHECK-VIS: ["neura.fmul_fmul"] -> ?? (I am not sure what the `dot` file's content look like though)
```